tests/easier68k/core/opcodes/test_opcode_helper.py

```diff
@@ -0,0 +1,40 @@
+"""
+Helper method for OPCODE unit tests.
+
+"""
+
+from easier68k.simulator.m68k import M68K
+from easier68k.core.enum.register import Register
+from easier68k.core.opcodes.opcode import Opcode
+from easier68k.core.models.memory_value import MemoryValue
+from easier68k.core.enum.condition_status_code import ConditionStatusCode
+
+
+def run_opcode_test(sim: M68K, opcode: Opcode, reg_check: Register, expected_val: int, program_increment: int, expected_ccr: list):
```

Comment:
I like this idea for quickly testing opcodes and the resulting CCRs.

Reply:
It's not ideal, but it works good enough for now. I'm planning on coming up with an easier way of setting up unit tests for OPCODEs, just not entirely sure how it should be done yet. Either way, A lot of the current testing files for OPCODEs can be refactored using this function.